ruby-challenges.rb

```diff
@@ -10,6 +10,14 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def filter_beverage(letter)
+  beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+  beverages_array.select { |i| i.include?(letter) }
+end
+
```

Comment:
This challenge is basically identical to one in the JavaScript section. If you put the array in the method you are limiting the functionality. Methods can take as many arguments as needed. The select statement looks great!